examples/custom_hint/src/main.rs

```diff
@@ -1,14 +1,18 @@
-use cairo_vm::cairo_run::cairo_run;
+#![allow(unused_imports)]
```

Comment:
Is this necessary?

Reply:
It was on [35afff0](https://github.com/lambdaclass/cairo-rs/pull/1258/commits/35afff024c5519e15161832edcab0de63d48a63b) (not currently)